src/Security/Items.cs

```diff
@@ -1051,6 +1051,20 @@ void SetValue (string value, IntPtr key)
 			}
 		}
 
+#if XAMCORE_2_0 && !WATCH && !TVOS
+		[iOS (9, 0), Mac (10, 11)]
+		public XamCore.LocalAuthentication.LAContext AuthenticationContext {
+			get {
+				return Fetch<XamCore.LocalAuthentication.LAContext> (SecItem.UseAuthenticationContext);
+			}
+			set {
+				if (value == null)
+					throw new ArgumentNullException (nameof (value));
```

Comment:
No according to the tests I did, while it does not crash the API, it just won't do what you expect it to do